pkg/controllers/vdb/init_db.go

```diff
@@ -114,7 +117,7 @@ func (g *GenericDatabaseInitializer) runInit(ctx context.Context) (ctrl.Result,
 	}
 
 	if res, err := g.initializer.execCmd(ctx, initiatorPod,
-		getHostList(podList)); verrors.IsReconcileAborted(res, err) {
+		getHostList(podList), genCommunalParmsMap(content)); verrors.IsReconcileAborted(res, err) {
```

Comment:
I wonder if it would be better to just use a string map from the start? Rather than build a string of key/value pairs, then parse the string into a map. It seems like a lot of processing we can save if we just started with a map at the beginning and build it up that way.